src/drive/web/modules/filelist/FileOpener.jsx

```diff
@@ -110,15 +109,47 @@ const FileOpener = ({
     )
   }
 
+  const isFolder = file =>
+    file.attributes && file.attributes.type === 'directory'
+  const isFile = file =>
+    (file.attributes && file.attributes.type === 'file') ||
+    file._type === 'io.cozy.files'
+  const isShortcut = file => file.class === 'shortcut'
+  const isNote = file => file.name.endsWith('.cozy-note')
+  let buildHref = ''
+  if (isFolder(file)) {
+    buildHref = `/#${folderUrlToNavigate(file.id)}`
+  } else if (isNote(file)) {
```

Comment:
All these helpers method should already been in https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/models/file.js#L36-L50 I think